packages/ketcher-core/src/application/editor/operations/monomer/index.ts

```diff
@@ -48,6 +48,19 @@ export class MonomerHoverOperation implements Operation {
     );
   }
 }
+export class APHoverOperation implements Operation {
```

Comment:
Actually, I don't understand the usage of "operation" and "command" regarding visual effect. Main purpose of commands is History, so that you can perform "undo" and "redo". Let's take an example: you hovered over a monomer, then you will create a command, so when you click "undo" the selection will be reset? 
The answer is – no. Commands should affect only model (chemical) changes, not hovers, selections, modals and any other visual effects.

Reply:
It is not my idea to use operation in such cases. If you look at this file, you will find similar approach there (MonomerHoverOperation). If we want to replace all this places, I think it is better to create a task further.

Reply:
Okay, I guess, we need to discuss it with @rrodionov91